src/main/java/com/google/api/generator/engine/ast/TypeNode.java

```diff
@@ -60,22 +68,10 @@ public static Builder builder() {
   }
 
   // TODO(miraleung): More type creation helpers to come...
-  public static TypeNode createReferenceType(Reference reference) {
+  public static TypeNode withReference(Reference reference) {
     return TypeNode.builder().setTypeKind(TypeKind.OBJECT).setReference(reference).build();
   }
 
-  public static TypeNode createReferenceArrayType(Reference reference) {
-    return TypeNode.builder()
-        .setTypeKind(TypeKind.OBJECT)
-        .setReference(reference)
-        .setIsArray(true)
-        .build();
-  }
-
```

Comment:
curious why we want to remove this method `createReferenceArrayType `?  it can be used to initialize STRING_ARRAY? Thanks

Reply:
There is currently no frequent need (aside from one-offs) for array creation. Removing unused / one-off methods is part of API best practices to help maintainability.